libs/cli/langchain_cli/package_template/pyproject.toml

```diff
@@ -7,11 +7,46 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-langchain = ">=0.0.313, <0.1"
-openai = "^0.28.1"
+langchain = ">=0.0.350, <0.2"
```

Comment:
why 0.2 instead of 0.1

Reply:
Given current plan thought it made sense to just update now but happy to adjust

Reply:
Reverting